model/host/host_test.go

```diff
@@ -568,6 +568,61 @@ func TestHostSetDNSName(t *testing.T) {
 	assert.Equal(t, newHostname, dbHost.Host, "existing hostname should be retained even if an empty string is passed")
 }
 
+func TestMarkReachable(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
The newer pattern is to use `t.Context()` instead of making a top-level context.